ansible/roles/network/tasks/main.yml

```diff
@@ -39,16 +43,86 @@
     dest: /boot/network.ini-sample
   tags:
     - touches_boot_partition
+  when: ansible_distribution_major_version|int != 9
 
 - name: Activate network manager
   command: systemctl enable screenly-net-manager.service chdir=/etc/systemd/system
+  when: ansible_distribution_major_version|int != 9
 
 - name: Activate network watchdog
   command: systemctl enable screenly-net-watchdog.timer chdir=/etc/systemd/system
+  when: ansible_distribution_major_version|int != 9
 
 - name: Install cronjob for watchdog
   cron:
     name: Screenly Net Watchdog
     minute: 0
     job: /usr/bin/python /usr/sbin/screenly_net_watchdog.py
     state: absent
+  when: ansible_distribution_major_version|int != 9
+
+
+# Use resin-wifi-connect if Stretch
+
+- name: Install NetworkManager
+  apt:
+    name: network-manager
+    state: present
+  when: ansible_distribution_major_version|int == 9
```

Comment:
`== 9` should be `>= 9`

Reply:
I alsk throught about it. but in the [script](https://github.com/resin-io/resin-wifi-connect/blob/master/scripts/raspbian-install.sh) it is strictly set to version 9 (although I suppose they could do `<=`). I  think that maybe it has reasons

Reply:
Well, basically we want to leave networking as is (i.e. remove our own network manager) for versions older than Stretch (9) and for Stretch and later, use WiFi Network Manager.